src/option_alpha/web/universe_routes.py

```
@@ -657,6 +660,71 @@ async def list_sectors(request: Request):
+    if _discovery_running:
+        raise HTTPException(status_code=409, detail="Discovery already running")
+
+    _discovery_running = True
+
+    async def _run_discovery():
+        global _discovery_running
+        conn = _get_conn(request)
+        try:
+            settings = request.app.state.settings
+            await run_discovery(conn, settings=settings)
+        except Exception as exc:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)